mappings/net/minecraft/block/entity/JigsawBlockEntity.mapping

```diff
@@ -25,3 +25,14 @@ CLASS net/minecraft/class_3751 net/minecraft/block/entity/JigsawBlockEntity
 			ARG 3 name
 		METHOD method_26401 byName (Ljava/lang/String;)Ljava/util/Optional;
 			ARG 0 name
+	CLASS class_5173 RuntimeStructurePiece
```

Comment:
Runtime?

Reply:
Hmm, `JigsawBlockEntity.Runtime` doesn't sound like a... structure piece

Reply:
I meant like what does `Runtime` mean in this case?

Reply:
a runtime structure piece. name from structure piece type